<launch>
  <arg name="mav_name" default="euroc6" />
  <arg name="room_name" default="LeoC6" />

  <group ns="$(arg mav_name)">

    <node name="trajectory_sampler" pkg="mav_planning_utils" type="trajectory_sampling_node" output="screen">
      <param name="publish_whole_trajectory" value="true" />
      <remap from="command/trajectory" to="eval_trajectory"/> 
    </node>

    <node name="task2_eval_node" pkg="euroc_task2" type="task2_eval_node" output="screen">
      <remap from="pose" to="vrpn_client/estimated_transform" />
      <remap from="saver_constraints_violated" to="/saver_constraints_violated" />
      <remap from="command/trajectory" to="eval_trajectory"/> 
    </node>

    <node name="task2_server_node" pkg="euroc_task2" type="task2_server_node" output="screen">
      <remap from="vrpn_client/pose" to="vrpn_client/estimated_transform" />
      <param name="file_path" value="res/wind.txt"/>
    </node>

  </group>

  <include file="$(find ros_vrpn_client)/launch/asl_vicon.launch">
    <arg name="object_name" value="$(arg mav_name)"/>
  </include>

  <include file="$(find mav_saver)/launch/mav_saver_real.launch">
    <arg name="mav_name" value="$(arg mav_name)"/>
  </include>

</launch>
